### ISSUE TITLE
Need help for compile

### ISSUE THREAD

**Issue by 4dvn** [spectran/CDJ-100S-MIDI-Adapter#1]
Can you write how to compile the source code on windows?

**Comment by spectran** [comment]
To compile it, you may use STM32CubeIDE (https://www.st.com/en/development-tools/stm32cubeide.html)
Simply import project to workspace, and build it.

**Comment by 4dvn** [comment]
Thank you, i need some image of pcb to wiring could you upload the image file of pcb?

**Comment by spectran** [comment]
I have used SprintLayout program to develop PCB. It generates .lay file, which can be found in the repository.
Now I will update repository with a .jpg image of PCB.

**Comment by 4dvn** [comment]
> I have used SprintLayout program to develop PCB. It generates .lay file, which can be found in the repository.
> Now I will update repository with a .jpg image of PCB.

Can i upload it to this board?
![image](https://user-images.githubusercontent.com/46510477/74085017-62743c00-4aa7-11ea-90b5-d74dcbcb84d9.png)

**Comment by spectran** [comment]
Yes, you can use any board with STM32F103C8 to upload firmware, but you need external transistors according to schematics.

**Comment by ScherbakovAl** [comment]
> To compile it, you may use STM32CubeIDE (https://www.st.com/en/development-tools/stm32cubeide.html)
> Simply import project to workspace, and build it.


I upload a project in CubeIde 1.5.1 using EGit. But I get an error when trying to build a project.

Description	Resource	Path	Location	Type
make: *** multiple target patterns.  Stop.	stm32f1xx_it.d	/CDJ_Control_C8/Debug/Src	line 1	C/C++ Problem

**Comment by ScherbakovAl** [comment]
The board is not detected when plugged into usb.  What should you pay attention to?

**Comment by spectran** [comment]
> 
> 
> The board is not detected when plugged into usb. What should you pay attention to?

If there is nothing when you connect the board to PC, try to check power nets, when D+ and D- nets of STM32F103. If the device is not recognized, check the voltage on D+ bus - it should be pulled to VDD. If the voltage is OK, check 8 MHz crystal.

**Comment by ScherbakovAl** [comment]
> > The board is not detected when plugged into usb. What should you pay attention to?
> 
> If there is nothing when you connect the board to PC, try to check power nets, when D+ and D- nets of STM32F103. If the device is not recognized, check the voltage on D+ bus - it should be pulled to VDD. If the voltage is OK, check 8 MHz crystal.

.. I added the following line to the code and the device was detected:
MX_USB_DEVICE_Init();
after -- "MX_GPIO_Init();"
